docs/annotations.md

```diff
@@ -89,3 +90,26 @@ metadata:
 spec:
   ...
 ```
+
+
+## Webhook Verification
+
+The following annotations can be used to configure the [Webhook Verification module](https://ngrok.com/docs/cloud-edge/modules/webhook).
+
+* `k8s.ngrok.com/webhook-validation-provider` - The webhook provider to use.
+* `k8s.ngrok.com/webhook-validation-secret-name` - The name of the Kubernetes Secret that contains the webhook secret.
+* `k8s.ngrok.com/webhook-validation-secret-key` - The key in the Kubernetes Secret that contains the webhook secret.
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: minimal-ingress
+  annotations:
+    k8s.ngrok.com/webhook-validation-provider: "github"
+    k8s.ngrok.com/webhook-validation-secret-name: "github-webhook-token"
+    k8s.ngrok.com/webhook-validation-secret-key: "SECRET_TOKEN"
```

Comment:
I think these should be called `verification` not `validation`. I see in the ngrok api go code and the docs page the type is an `EndpointWebhookValidation` but the actual key of the body is verification. I think before edges, when we used endpoint configurations, this was the name, but got changed to verification in the edge world 
https://github.com/ngrok/ngrok-api-go/blob/da8b5e25f3a1272c3106bfb06b05fd3847fa201c/datatypes.go#L2426

We may even go as far as naming these the same way the ngrok cli is named such as `https-edge-route-webhook-verification`

Reply:
Fixed in [43b3cf3](https://github.com/ngrok/kubernetes-ingress-controller/pull/122/commits/43b3cf3d377e5ed2b9983e1fa3072c738d467f8d)